docs/src/testing/testing.md

```diff
@@ -12,13 +12,17 @@ fn sum(a: felt252, b: felt252) -> felt252 {
     return a + b;
 }
 
-#[test]
-fn test_sum() {
-    assert(sum(2, 3) == 5, 'sum incorrect');
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn test_sum() {
+        assert(sum(2, 3) == 5, 'sum incorrect');
+    }
 }
 ```
 
 It is a common practice to keep your unit tests in the same file as the tested code. 
+Keep in mind that all tests in `src` folder have to be in a module annotated with `#[cfg(test)]`.
```

Comment:
@cptartur this is the mention you asked for, the rest is just output change so that it can happen in reality (adding `tests::` prefix for tests in `src` etc.)